### PR TITLE
Fix for UI issues when creating new API Version

### PR DESCRIPTION
**Pull request by sanjulamadurapperuma**
Fix for UI issues when creating new API Version and improving Tooltip functionality.

Fixes [wso2/product-apim#5610](https://github.com/wso2/product-apim/issues/5610)

NewVersionUI

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json [hunk 1]
```diff
@@ -520,16 +520,14 @@
...
   "Apis.Details.NewVersion.NewVersion.create": "Create",
   "Apis.Details.NewVersion.NewVersion.create.new.version": "Create New Version",
   "Apis.Details.NewVersion.NewVersion.default": "Make this the default version",
   "Apis.Details.NewVersion.NewVersion.error": "Something went wrong while creating a new version!. Error:",
-  "Apis.Details.NewVersion.NewVersion.helper.text": "{helper}",
   "Apis.Details.NewVersion.NewVersion.new.version": "New Version",
   "Apis.Details.NewVersion.NewVersion.success": "Successfully created new version",
-  "Apis.Details.NewVersion.NewVersion.tooltip": "Marks one API version in a group as the default so that it can be invoked without specifying the version number in the URL. For example, if you mark http://host:port/youtube/2.0 as the default API, requests made to http://host:port/youtube/ are automatically routed to version 2.0. If you mark an unpublished API as the default, the previous default published API will still be used as the default until the new default API is published.",
+  "Apis.Details.NewVersion.NewVersion.tooltip": "Determine if this should be the default version among multiple versions of the API",
```
Comment (bhathiya) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json]: @shaniR should this be "determines"?

Comment (shaniR) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json]: How about rephrasing it as "Indicates if this is the default version of the API. If an API is invoked without specifying a version, the API Gateway will route the request to the default version of the API. "

Comment (sanjulamadurapperuma) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json]: > How about rephrasing it as "Indicates if this is the default version of the API. If an API is invoked without specifying a version, the API Gateway will route the request to the default version of the API. "

@bhathiya What do you think?

Comment (bhathiya) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json]: +1

Comment (sanjulamadurapperuma) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/en.json]: Updated Tooltip description as suggested. Thanks.

**Pull request merged**